libraries/kafka_config.rb

```diff
@@ -34,7 +34,8 @@ class KafkaConfig < Chef::Resource
                     'kafka.request.logger' => 'WARN',
                     'kafka.controller' => 'TRACE',
                     'kafka.log.LogCleaner' => 'INFO',
-                    'state.change.logger' => 'TRACE'
+                    'state.change.logger' => 'TRACE',
+                    'kafka.authorizer.logger' => 'WARN'
```

Comment:
do you want me to update https://github.com/bloomberg/kafka-cookbook/blob/master/attributes/default.rb#L44 ?  That means I need to update the checksum on https://github.com/bloomberg/kafka-cookbook/blob/master/attributes/default.rb#L46, right?  

But it looks like the mirror defined on https://github.com/bloomberg/kafka-cookbook/blob/master/attributes/default.rb#L47 no longer works.